packages/lib-react-components/src/ZooHeader/components/SignedInUserNavigation/SignedInUserNavigation.js

```diff
@@ -1,49 +1,112 @@
-import counterpart from 'counterpart'
-import { Box } from 'grommet'
-import { MailOption, Notification } from 'grommet-icons'
 import PropTypes from 'prop-types'
 import React from 'react'
+import { Box } from 'grommet'
 
+import zooTheme from '@zooniverse/grommet-theme'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faBell as fasBell, faEnvelope as fasEnvelope } from '@fortawesome/free-solid-svg-icons'
+import { faBell as farBell, faEnvelope as farEnvelope } from '@fortawesome/free-regular-svg-icons'
+
+import counterpart from 'counterpart'
 import en from './locales/en'
+
+import NarrowMainNavMenu from '../NarrowMainNavMenu'
 import NavListItem from '../NavListItem'
 import UserMenu from '../UserMenu'
 import { getHost } from '../../helpers'
 
 counterpart.registerTranslations('en', en)
 
-export default function SignedInUserNavigation ({ host, screenWidth, signOut, user }) {
-  const notifications = (screenWidth === 'wide')
-    ? <NavListItem label={counterpart('SignedInUserNavigation.navListLabels.notifications')} url={`${host}/notifications`} />
-    : <Notification />
-
-  const messages = (screenWidth === 'wide')
-    ? <NavListItem label={counterpart('SignedInUserNavigation.navListLabels.messages')} url={`${host}/inbox`} />
-    : <MailOption />
-
-  return (
-    <Box
-      align='center'
-      direction='row'
-      tag='nav'
-    >
-      {notifications}
-      {messages}
-      <UserMenu
-        signOut={signOut}
-        user={user}
-      />
-    </Box>
-  )
+export default function SignedInUserNavigation (props) {
+  const {
+    adminNavLinkLabel,
+    adminNavLinkURL,
+    host,
+    isAdmin,
+    isNarrow,
+    mainHeaderNavListLabels,
+    mainHeaderNavListURLs,
+    unreadMessages,
+    unreadNotifications,
+    signOut,
+    user
+  } = props
+
+  const notificationLabelString = (unreadNotifications)
+    ? `${counterpart('SignedInUserNavigation.navListLabels.notifications')} (${unreadNotifications})`
+    : counterpart('SignedInUserNavigation.navListLabels.notifications')
```

Comment:
Instead of having a ternary doing nearly the same thing twice, you could do a string interpolation in counterpart:

```
// en.json
...
notifications: 'Notifications %(count)s'
...

// component
const notificationLabelString = counterpart('SignedInUserNavigation.navListLabels.notifications', {
  count: (unreadNotifications) ? `(${unreadNotifications})` : ''
})
```

Reply:
You can also do away with the ternary completely by defining a string in counterpart for the case where the count is 0: https://github.com/zooniverse/Panoptes-Front-End/blob/d6fda2f400f53b71bf05e8555c5be9f317e7f8c5/app/locales/en.js#L75-L79

Reply:
Much better \o/

Reply:
https://github.com/zooniverse/front-end-monorepo/pull/290